src/System.IO.FileSystem.Watcher/src/System.IO.FileSystem.Watcher.csproj

```diff
@@ -58,19 +58,19 @@
     <Compile Include="$(CommonPath)\Interop\Unix\Interop.Libraries.cs">
       <Link>Common\Interop\Unix\Interop.Libraries.cs</Link>
     </Compile>
-    <Compile Include="$(CommonPath)\CoreLib\Interop\Unix\System.Native\Interop.Read.cs">
+    <Compile Include="$(CoreLibDir)Interop\Unix\System.Native\Interop.Read.cs">
       <Link>Common\Interop\Unix\libc\Interop.Read.cs</Link>
```

Comment:
Ditto. I guess we don't want to fix these? There are many like this, seem to be out of the scope of this PR?